File: Modulo4-ObjectiveC-Lorena/Exercicio5/Exercicio5.m
//
//  main.m
//  Exercicio5-C
//
//  Created by Lorena Rodrigues Bruno on 10/09/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "PontoCartesiano.h"

@implementation PontoCartesiano
@synthesize pontoX, pontoY;
-(id)initPonto:(int)pontoX Segundo:(int)pontoY {
    self = [super init];
    self.pontoX = pontoX;
    self.pontoY = pontoY;
    return self;
}

@end

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        PontoCartesiano *ponto = [[PontoCartesiano alloc] initPonto:15 Segundo:10];
        printf("Ponto cartesiano: (%i, %i)\n", [ponto pontoX], [ponto pontoY]);
    }
    return 0;
}

File: Modulo4-Lorena/Exercicio5.swift
//
//  main.swift
//  Exercicio5
//
//  Created by Lorena Rodrigues Bruno on 28/08/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

import Foundation

class Ponto {
    var eixoX = Int()
    var eixoY = Int()
    
    init(eixoX : Int, eixoY : Int) {
        self.eixoX = eixoX
        self.eixoY = eixoY
    }
    
    func imprimeCoordenada() {
        print("Coordenadas do ponto: (\(eixoX),\(eixoY)) ")
    }
    
}

//teste
//var ponto1 = Ponto(eixoX: 3, eixoY: 5)
//ponto1.imprimeCoordenada()
